mp_cnn/dataset.py

```diff
@@ -26,193 +24,38 @@
 logger.addHandler(ch)
 
 
-class DatasetType(Enum):
-    TRAIN = 1
-    TEST = 2
-    DEV = 3
+unknown_vec = None
```

Comment:
Why do you need unknown_vec here? I am a little confused.

Reply:
I want to use the same unknown vector for all OOV words' vectors. In torchtext I must pass in a function that when given a tensor `t`, return a tensor of the same size as `t`. In the function below I fix a vector the first the function is run so for later runs I get the same vector.

```python
 +def unk(tensor):
 +    global unknown_vec
 +    if unknown_vec is None:
 +        unknown_vec = torch.Tensor(tensor.size())
 +        unknown_vec.normal_(0, 0.01)
 +    return unknown_vec
```

Please let me know if there is a better way.

Reply:
I see. It is reasonable if you want to use a static variable in python like this. But people prefer to avoid using global variable in python. https://stackoverflow.com/questions/279561/what-is-the-python-equivalent-of-static-variables-inside-a-function may provide a better solution if you do not share a global variable with other modules.

Reply:
I will fix this in the next update to the PR